frontend/common/crud/useGet.tsx

```diff
@@ -12,9 +12,28 @@ export function useGet<T>(
   swrConfiguration: SWRConfiguration = {}
 ) {
   const getRequest = useGetRequest<T>();
+  const [lastValue, setLastValue] = useState<T | undefined>();
+  const getRequestWithoutAbortError = useCallback(
+    async (url: string) => {
+      try {
+        return await getRequest(url).then((value) => {
+          setLastValue(value);
+          return value;
+        });
+      } catch (error) {
+        if (error instanceof Error && error.name === 'AbortError') {
+          // We need to return the last value if the request was aborted
+          // This is because SWR internal workings
+          return lastValue as T;
```

Comment:
@jamestalton Do we need to return a default value different than `undefined`?

Reply:
I am seeing a case where two swr calls are going on at the same time.
One aborts, the other then gets undefined.
Not sure what to do about it. returning the last value works, but i am worried it might be out of date.
Still pondering this one.